plugins.json

```diff
@@ -1890,7 +1890,7 @@
         "author": "greyaz",
         "compatible_version": ">=1.2.8",
         "description": "A 'task-first' and high-quality theme for Kanboard. It's aimed at better mobile experiences, common plugins' compatibilities, and customization friendly.",
-        "download": "https://github.com/greyaz/ThemeRevision/releases/download/1.1.9/ThemeRevision.zip",
+        "download": "https://github.com/JustFxDev/ThemeMaestro/raw/main/release/v0.7.0/ThemeMaestro.zip",
```

Comment:
I would have used this link, but if yours works, it works.

`https://github.com/JustFxDev/ThemeMaestro/releases/download/v0.7.0/ThemeMaestro_v0.7.0.zip`